internal/store/elasticsearch/discovery_search_repository_test.go

```diff
@@ -194,6 +194,17 @@ func TestSearcherSearch(t *testing.T) {
 					{Type: "table", AssetID: "bigquery::gcpproject/dataset/tablename-common"},
 				},
 			},
+			{
+				Description: "should return 'bigquery::gcpproject/dataset/tablename-common' resource on top if searched for text 'tablename-common'",
```

Comment:
This test passes even without the source code changes.

Reply:
fixed